sbtc/src/error.rs

```diff
@@ -10,6 +10,9 @@ pub enum Error {
     /// Error when creating an RPC client to bitcoin-core
     #[error("could not create RPC client to {1}; {0}")]
     BitcoinCoreRpcClient(#[source] bitcoincore_rpc::Error, String),
+    /// Error when using a BitcoinClient trait function
```

Comment:
Don't take any action on this, but just for my own curiosity why have these in the `sbtc` crate instead of the `signer` crate?

Reply:
Good question. I figured Emily would want to use this functionality as well when validating deposit requests so I put the bitcoin client trait here. There is probably a better way to organize this all though.